src/lasp_core.erl

```diff
@@ -361,7 +361,7 @@ update(Id, Operation, Actor, Store) ->
     {ok, var()} | not_found().
 update(Id, Operation, Actor, MetadataFun, Store) ->
     {ok, #dv{value=Value0, type=Type}} = do(get, [Store, Id]),
-    {ok, Value} = lasp_type:update(Type, Operation, Actor, Value0),
+    {ok, Value} = lasp_type:update(Type, Operation, {Id, Actor}, Value0),
```

Comment:
I copied my comment in types PR.

In order to handle the union()/intersection()/product() operations properly, the actor identifier was not enough. Therefore, I created the EventId which is {ObjectId, ActorId}. If this is not allowed, then I have to find another way for it since this is an essential requirement for those operations.